### PR TITLE
Add several keyboards made by kibou

### PR DESCRIPTION
**Pull request by biuboom-jun**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/winter/rules.mk
[the remark this answers is not in the document]
Comment (biuboom-jun) [reply keyboards/winter/rules.mk:1]: Hello, after I modified this template, I got an error "File 'keyboards/winter/rules.mk' Requires Formatting
Error: Requires Formatting", this error also occurs in my info.json and readme.md, can you give me a solution?

[the remark this answers is not in the document]
Comment (zvecr) [reply keyboards/winter/rules.mk:1]: You need to use unix line endings.

Comment (zvecr) [inline keyboards/winter/rules.mk:1]: This file can not be empty

**Comment by tzarc** [comment]
As per the PR Checklist this should be multiple PRs, one keyboard each.

**Pull request closed without merging**